addons/stock/wizard/stock_quant_reserve.py

```diff
@@ -0,0 +1,77 @@
+from odoo import _, api, Command, fields, models
+from odoo.exceptions import UserError
+
+
+class ReserveStockQuantLine(models.TransientModel):
+    _name = 'stock.quant.reserve.line'
+    _description = 'Choose quantity to reserve from each stock quant'
+
+    quant_id = fields.Many2one('stock.quant', required=True, readonly=True)
+    chosen = fields.Boolean()
```

Comment:
This is a slightly cleaner name that is more consistent with our usual boolean field names

```suggestion
    is_selected = fields.Boolean()
```